pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -2530,6 +2530,50 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 		})
 	})
 
+	Context("with memoryMode", func() {
+		var vmi *v1.VirtualMachineInstance
+		BeforeEach(func() {
+			vmi = api.NewMinimalVMI("testvmi")
+			memMode := v1.MemoryModePreferred
+			vmi.Spec.Domain.CPU = &v1.CPU{
+				DedicatedCPUPlacement: true,
+				NUMA: &v1.NUMA{
+					MemoryMode: &memMode,
+				},
+			}
+			enableFeatureGate(virtconfig.NUMAFeatureGate)
+		})
+		It("should fail when NUMA feature gate is disabled", func() {
```

Comment:
nit: can you please add another newline between `It`s?

Reply:
👍